classes/class-wpcom-liveblog-entry-query.php

```diff
@@ -142,11 +142,6 @@ public static function remove_replaced_deleted_entries( $entries = array(), $num
 				unset( $entries_by_id[ $id ] );
 			}
 
-			// Remove deleted entries
-			if ( 'delete' === $entry->get_type() ) {
-				unset( $entries_by_id[ $id ] );
-			}
-
```

Comment:
I'll have a look at the branch I didn't know there was work being done on this, is it related to #108?